ballerina/tests/10-custom-datetime-test.bal

```diff
@@ -34,26 +36,28 @@ isolated function beforeInsertTimeFunc() returns sql:Error? {
        PRIMARY KEY(PK)
        )`
    );
-
    check oracledbClient.close();
 }
 
 @test:Config {
-   groups:["execute"]
+   groups:["datetime"]
 }
 isolated function insertIntervalWithString() returns sql:Error? {
    Client oracledbClient = check new(HOST, USER, PASSWORD, DATABASE, PORT);
    string date = "05-JAN-2020 10:35:10 AM";
+   string dateOnly = "05-JAN-2020";
+   string timeOnly = "0 11:00:00";
```

Comment:
Shall we define new ballerina types to pass intervals?

Reply:
yes new types are defined. 

https://github.com/ballerina-platform/module-ballerinax-oracledb/blob/2916a3187a51fba7241f8e43ccdbfc5a15c4cec5/ballerina/types.bal#L25

https://github.com/ballerina-platform/module-ballerinax-oracledb/blob/2916a3187a51fba7241f8e43ccdbfc5a15c4cec5/ballerina/types.bal#L37

Reply:
AFAIU, In oracle DB, we can define optional precision values for years, months, days, fractions of seconds when setting the value. we need to support that as well

Reply:
As discussed, shall we create a new issue for interval support. I think we need to relook at the interval support separately

Reply:
Issue created : https://github.com/ballerina-platform/ballerina-standard-library/issues/1763